mllib/src/test/scala/org/apache/spark/mllib/evaluation/RegressionMetricsSuite.scala

```diff
@@ -22,91 +22,115 @@ import org.apache.spark.mllib.util.MLlibTestSparkContext
 import org.apache.spark.mllib.util.TestingUtils._
 
 class RegressionMetricsSuite extends SparkFunSuite with MLlibTestSparkContext {
+  val obs = List[Double](77, 85, 62, 55, 63, 88, 57, 81, 51)
+  val eps = 1E-5
 
   test("regression metrics for unbiased (includes intercept term) predictor") {
```

Comment:
I think @iyounus made up a new test that is then used for both cases. Yeah it'd be kind of good to see that the old test still passes, but this is pretty reliably benchmarked vs R now and is a slightly better test.